src/utils/account.ts

```diff
@@ -1,43 +1,64 @@
 import { Keypair, Mnemonic, Address } from '@helium/crypto-react-native'
 import * as SecureStore from 'expo-secure-store'
 
+type AccountStoreKey = BooleanKey | StringKey
+
+const stringKeys = ['mnemonic', 'keypair', 'address', 'userPin'] as const
+type StringKey = typeof stringKeys[number]
+
+const boolKeys = [
+  'accountBackedUp',
```

Comment:
do we need these registered in `account.js` if they're also represented in redux?

Reply:
I like them being here even though it's a little redundant. The use case is when a user deletes and reinstalls the app. The keystore will be persisted whereas async-storage/redux-persist won't be.

Reply:
ok that works